kubernetes-openapi-client/README.md

```diff
@@ -1,8 +1,49 @@
-# kubernetes-watch-client
-
-Client for streaming events from watch enabled endpoints.
+# kubernetes-openapi-client
 
 ## Example
+
+```haskell
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+import           Data.Function           ((&))
+import qualified Kubernetes.OpenAPI.API.CoreV1
+import           Kubernetes.OpenAPI.Client       (dispatchMime)
```

Comment:
@guoshimin I don't think the README example here reflects the lates module name changes.